en/ability.json

```
@@ -433,103 +433,103 @@
   },
   "unaware": {
     "name": "Unaware",
-    "description": "This Pokémon ignores the opposing Pokémon’s stat changes."
+    "description": "The Pokémon ignores the opposing Pokémon’s stat changes when attacking or defending."
   },
   "tintedLens": {
     "name": "Tinted Lens",
-    "description": "The Pokémon can use \"not very effective\" moves to deal regular damage."
+    "description": "The Pokemon’s attacks that are not very effective on a target deal double damage."
   },
   "filter": {
     "name": "Filter",
-    "description": "Reduces the power of supereffective attacks taken."
+    "description": "Reduces the power of supereffective attacks taken by 25%."
   },
   "slowStart": {
     "name": "Slow Start",
     "description": "For five turns, the Pokémon’s Attack and Speed stats are halved."
   },
   "scrappy": {
     "name": "Scrappy",
-    "description": "The Pokémon can hit Ghost-type Pokémon with Normal- and Fighting-type moves."
+    "description": "The Pokémon can hit Ghost-type Pokémon with Normal- and Fighting-type moves. It is also unaffected by Intimidate."
   },
   "stormDrain": {
     "name": "Storm Drain",
-    "description": "Draws in all Water-type moves. Instead of being hit by Water-type moves, it boosts its Sp. Atk."
+    "description": "The Pokémon draws in all Water-type moves. Instead of taking damage from them, its Sp. Atk stat is boosted."
   },
   "iceBody": {
     "name": "Ice Body",
-    "description": "The Pokémon gradually regains HP in snow."
+    "description": "The Pokémon gradually regains 1/16 of their maximum HP in snow or hail. It also takes no damage from hail."
   },
   "solidRock": {
     "name": "Solid Rock",
-    "description": "Reduces the power of supereffective attacks taken."
+    "description": "Reduces the power of supereffective attacks taken by 25%."
   },
   "snowWarning": {
     "name": "Snow Warning",
     "description": "The Pokémon makes it snow when it enters a battle."
   },
   "honeyGather": {
     "name": "Honey Gather",
-    "description": "The Pokémon gathers Honey after a battle is won. The Honey is then sold for money."
+    "description": "The Pokémon gathers Honey which is then sold for money after a battle is won, even if it did not participate."
   },
   "frisk": {
     "name": "Frisk",
     "description": "When it enters a battle, the Pokémon can check an opposing Pokémon’s Ability."
   },
   "reckless": {
     "name": "Reckless",
-    "description": "Powers up moves that have recoil damage."
+    "description": "Powers up moves that have recoil damage by 20%. Does not include Struggle."
```

```suggestion
    "description": "Powers up moves that have recoil damage by 20%."
```
I don't think we need to mention Struggle because the user's abilities don't affect it at all.